regparser/notice/compiler.py

```diff
@@ -192,7 +192,11 @@ def add_child(children, node, order=None):
             sort_order = sort_order[:-1]
             # Use bisect so the whole list isn't resorted (the original list
             # may not be strictly sorted)
-            insert_idx = bisect(sort_order, new_el_sort)
+            try:
+                insert_idx = bisect(sort_order, new_el_sort)
+            except TypeError:
+                logger.warn("Could not add child to tree: %s", node)
+                return children
```

Comment:
Wouldn't this result in half finished parsing? Seems like it should error loudly.